datasets/objectenopenbareruimte/dataset.json

```diff
@@ -13,263 +12,601 @@
   "title": "Beheerobjecten openbare ruimte",
   "language": "nl",
   "dateCreated": "2022-06-01T00:00:00+01:00",
-  "license": "Gesloten",
-  "reasonsNonPublic": [
-    "5.1 2f: Zwaarwegend belang: vertrouwelijke of concurrentiegevoelige bedrijfs- en fabricagegegevens"
-  ],
-  "accrualPeriodicity": "dagelijks",
+  "license": "openbaar, tenzij anders aangegeven / behoudens uitzonderingen",
   "creator": "Datateam Beeldschoon/BOR",
   "publisher": "Datateam Beheer en Openbare Ruimte",
   "description": "Objecten in de openbare ruimte die beheerd worden door de Gemeente Amsterdam",
   "status": "beschikbaar",
   "keywords": [
-    "asset",
-    "verhardingen",
-    "groenobjecten",
-    "terreindeel"
+      "asset",
```

Comment:
Er gaat iets mis met het inspringen. De schemas horen een indent van 2 spaties te hebben, maar het zijn er nu 4 geworden. Daardoor krijgen we onleesbare diffs. Is het mogelijk de editor zo in te stellen dat het er 2 spaties worden gebruiken. En hiermee dan een nieuwe commit aan de PR toe te voegen? (ik kan wel even assisteren als het nodig is).

Reply:
Anders kun je dit vanaf de command line doen:

```shell
sudo apt install jq moreutils
jq . dataset.json | sponge dataset.json
```

Reply:
ik werk helaas nog vanuit Windows  laptop.

Reply:
Dit heb ik al opgelost via .editorconfig extension.